Meta/gn/secondary/AK/BUILD.gn

```diff
@@ -11,16 +11,18 @@ source_set("AK") {
   public_configs = [ ":ak_headers" ]
   public_deps = [ ":ak_debug_gen" ]
 
-  # NOTE: Headers only!
+  # FIXME: Split out non-kernel sources to their own set
```

Comment:
AK is still a source set here, which means everyone will get the object files linked in. Probably not quite what we need.

Reply:
Ah good catch. I'm guessing we'll have to play with it once Serenity is building with GN, but I've made it a shared lib now.